portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -16,33 +16,56 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
  
 /** Servlet that adds comments to datastore */
 @WebServlet("/data") 
 public class DataServlet extends HttpServlet 
 { 
+  public void init() throws ServletException {
+    try {
+        languageService = LanguageServiceClient.create();
+    } catch (Exception e) {
```

Comment:
Since this is a broad exception catching clause, are we always guaranteed that an I/O exception has occurred? Maybe the error message should be a bit more generic.

Reply:
I'll change the error message